packages/docs/getting-started/running-a-validator.md

```diff
@@ -117,177 +133,170 @@ In all the commands we are going to see the `CELO_IMAGE` variable to refer to th
-
-You can find the complete instructions about how to run the [Attestation Service at the documentation page](running-attestation-service.md).
-
-### Stop the containers
-
-You can stop the Docker containers at any time without problem. If you stop your containers that means those containers stop of providing service.
-The data dir of the validator and the proxy are Docker volumes mounted in the containers from the `celo-data-dir` you created at the very beginning. So if you don't remove that folder, you can stop or restart the containers without losing any data.
-
-You can stop the `celo-validator` and `celo-proxy` containers running:
+It will prompt you for the passphrase you've chosen for the Validator signer key. Let's save the resulting proof-of-possession to three environment variables:
 
 ```bash
-docker stop celo-validator celo-proxy
+# On your local machine
+export CELO_VALIDATOR_SIGNER_POP=<YOUR-VALIDATOR-SIGNER-POP>
+export CELO_VALIDATOR_BLS_PUBLIC_KEY=<YOUR-VALIDATOR-BLS-PUBLIC-KEY>
+export CELO_VALIDATOR_BLS_SIGNATURE=<YOUR-VALIDATOR-BLS-SIGNATURE>
 ```
 
-And you can remove the containers (not the data dir) running:
+Let's stop the node from running for now:
 
 ```bash
-docker rm -f celo-validator celo-proxy
+# On the validator machine
+docker stop celo-validator
+docker rm celo-validator
 ```
 
-### Running the Docker containers in the background
+We'll get back to this machine later, but for now, let's give it a proxy.
 
-There are different option for executing Docker containers in the background. The most typical one is to use in your docker run commands the `-d` option. Also for long running processes, specially when you run in a remote computer, you can use a tool like [screen](https://ss64.com/osx/screen.html). It allows to connect and disconnect from running processes providing an easy way to manage long run processes.
+### Deploy a proxy
 
-It's out of the scope of this documentation to go through the `screen` options, but you can use the following command format with your `docker` commands:
+To avoid exposing the validator to the public internet, we are deploying a proxy node which is responsible to communicate with the network. On our Proxy machine, we'll setup the node as per usual now:
 
 ```bash
-screen -S <SESSION NAME> -d -m <YOUR COMMAND>
+# On the proxy machine
+mkdir -p celo-data-dir/proxy
+cd celo-data-dir/proxy
+docker run -v $PWD/proxy:/root/.celo $CELO_IMAGE init /celo/genesis.json
+docker run -v $PWD/proxy:/root/.celo --entrypoint cp $CELO_IMAGE /celo/static-nodes.json /root/.celo/
 ```
 
-For example:
+You can then run the proxy with
 
 ```bash
-screen -S celo-validator -d -m docker run --name celo-validator --restart always -p 127.0.0.1:8545:8545 .......
+# On the proxy machine
```

Comment:
\# Note that you'll have to export CELO_VALIDATOR__SIGNER_ADDRESS on this machine